modular_citadel/code/game/objects/items/balls.dm

```diff
@@ -0,0 +1,60 @@
+/obj/item/toy/tennis
+	name = "tennis ball"
+	desc = "A classical tennis ball. It appears to have faint bite marks scattered all over its surface."
+	icon = 'modular_citadel/icons/obj/balls.dmi'
+	icon_state = "tennis_classic"
+	lefthand_file = 'modular_citadel/icons/mob/inhands/balls_left.dmi'
+	righthand_file = 'modular_citadel/icons/mob/inhands/balls_right.dmi'
+	item_state = "tennis_classic"
+	alternate_worn_icon = 'modular_citadel/icons/mob/mouthball.dmi'
+	slot_flags = SLOT_HEAD
```

Comment:
If you want to make this equippable in more than just the head slot, check out how headphones handle it here https://github.com/Citadel-Station-13/Citadel-Station-13/blob/master/code/modules/clothing/ears/_ears.dm#L20

Reply:
The thing is, it simply refuses to work properly in the mask slot. Even the icon_state breaks when it is placed in the mask slot for whatever reason.